data/tasks.txt

```diff
@@ -1,5 +1,4 @@
-E | 0 | capt day | 2020-12-03, 15:30
```

Comment:
Consider adding this data file to your `.gitignore`, so that it isn't uploaded to the repo.